v4.5/Makefile

```diff
@@ -0,0 +1,322 @@
+PROFILE := v4.5
```

Comment:
It's generated from the DLLs in this folder, which iirc we created around the Mono 4.0 timeframe so whatever we had implemented back then.

Updating the older profiles with the APIs we implemented/added by now should be a different task IMO.